.pep8

```diff
@@ -0,0 +1,2 @@
+[pep8]
+max_line_length=119
```

Comment:
note to reviewers: 
I chose 119 as that was what is currently in the django IDA cookie-cutter which is the closest that we have to a python template that I can think of right now.